chart/compass/templates/internal-communication-policies.yaml

```diff
@@ -67,6 +67,9 @@ spec:
     - from: # or anything other only if the request is authenticated via RequestAuthentication
         - source:
             requestPrincipals: ["*"]
+    - from: # or anything that's using the Prometheus service account
```

Comment:
move the metrics path "to" embedded in this rule so that we allow traffic only from prom to them

Reply:
I'm not sure whether we can have such a property inside the `from` rule. I've checked the [Istio docs](https://istio.io/latest/docs/reference/config/security/authorization-policy/#Source) and I don't think that this approach is feasible.